File: Modul-3/Soal 5/PRAK305-2410817110009-MuhammadAlfiGunawan.c
#include <stdio.h>

int main (){
    int detik1, detik2, menit, jam, hari;

    printf("Masukkan Detik: ");
    scanf("%d", &detik1);

    hari = detik1/86400;
    jam = (detik1 % 86400) /3600;
    menit = (detik1 % 3600)/60;
    detik2 = detik1 % 60;

    if (hari >= 1){
        printf("%d hari %.2d:%.2d:%.2d\n", hari,jam,menit,detik2);
        }
    else{
        printf("%.2d:%.2d:%.2d\n", jam,menit,detik2);
        }


}

File: Modul-3/Soal 5/PRAK305-2410817110009-MuhammadAlfiGunawan.py
detik1 = int(input("Masukkan Detik: "))

hari = int(detik1/86400)
jam = int((detik1 % 86400) /3600)
menit = int((detik1 % 3600)/60)
detik2 = int(detik1 % 60)

if hari >= 1:
    print(f"{hari} hari {jam:02}:{menit:02}:{detik2:02}")
else:
    print(f"{jam:02}:{menit:02}:{detik2:02}")